src/index.ts

```diff
@@ -3,10 +3,12 @@ import { ApplicationConfig } from '@loopback/core'
 
 const logger = require('./services/logger')
 
+const DEFAULT_APPLICATION_OPTIONS = { load: true }
+
 export { PocketGatewayApplication }
 
 export async function main(options: ApplicationConfig = {}): Promise<PocketGatewayApplication> {
-  options.env = { load: true }
+  options.env = DEFAULT_APPLICATION_OPTIONS
```

Comment:
Haha this is okay, but I think we could go further; I meant assigning a _default value_ to the `options` parameter. :slightly_smiling_face: